cli/command/container/create.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"fmt"
 	"io"
+	"io/ioutil"
```

Comment:
oh! this needs to be updated to use `io` now (as `io/ioutil` has been deprecated)